vue-handler.js

```diff
@@ -155,13 +155,13 @@ function vueFilePage(data) {
                 </div>
                 <div v-if="entry.type == 'mp3' || entry.type == 'm4a'">
                     <audio controls :src="webUrl +  entryId + '/download'" preload="auto">
-                        Your browser does not support the <code>audio</code> element.
+                        ඔබගේ අතිරික්සුව <code>audio</code> අංගය සඳහා සහාය නොදක්වයි.

```

Comment:
browser inside brackets needed 🤣

Reply:
Agree. 
Or can use බ්රවුසරය, which is the common word used in Sinhala.